torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -520,6 +520,7 @@ RegisterOperators reg({
             };
           }
         }),
+    Operator("aten::_unwrap_optional(t? optional) -> t", noop),
```

Comment:
Since this is a noop won't the type refinement happen every time an optional is assigned to another type (regardless of whether `_unwrap_optional` was used?)